### PR TITLE
readme: update supported features table

### PR DESCRIPTION
**Pull request by sam-golioth**
Remove experimental features (FW compression, delta updates, remote shell)

Add Zephyr column

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,24 +96,21 @@ The following table lists which SDK features (rows) are support for each platfor
...
+| --- | --- | --- | --- | --- |
+| OTA FW Update | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: <sup>1</sup> | :heavy_check_mark: |
+| Cloud Logging | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| LightDB State | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| LightDB Stream | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Remote Procedure Call | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Settings | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| CoAP/DTLS Client | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| PSK Auth | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Certificate Auth | :heavy_check_mark: | :soon: | :heavy_check_mark: | :heavy_check_mark: |
```
Comment (mniestroj) [inline README.md:109]:
```suggestion
| Certificate Auth | :heavy_check_mark: | :soon: | :heavy_check_mark: | :soon: |
```

File: README.md [hunk 2]
```diff
@@ -96,24 +96,21 @@ The following table lists which SDK features (rows) are support for each platfor
...
-| OTA FW compression | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| OTA FW delta update | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| Terminal shell | :heavy_check_mark: | :soon: | :x: |
-| Remote Shell | :heavy_check_mark: | :soon: | :soon: |
-| Serial Provisioning | :heavy_check_mark: | :soon: | :x: |
-| BLE Provisioning | :heavy_check_mark: | :soon: | :x: |
+| Feature | ESP-IDF | ModusToolbox | Linux | Zephyr |
+| --- | --- | --- | --- | --- |
+| OTA FW Update | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: <sup>1</sup> | :heavy_check_mark: |
+| Cloud Logging | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
```
Comment (mniestroj) [inline README.md:102]:
```suggestion
| Cloud Logging | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :soon: |
```

File: README.md [hunk 3]
```diff
@@ -96,24 +96,21 @@ The following table lists which SDK features (rows) are support for each platfor
...
+| LightDB State | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| LightDB Stream | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Remote Procedure Call | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Settings | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| CoAP/DTLS Client | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| PSK Auth | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Certificate Auth | :heavy_check_mark: | :soon: | :heavy_check_mark: | :heavy_check_mark: |
+| Sign+Verify OTA images | :soon: | :heavy_check_mark: | :soon: | :soon: |
+| Terminal shell | :heavy_check_mark: | :soon: | :x: | :heavy_check_mark: |
```
Comment (mniestroj) [inline README.md:111]:
```suggestion
| Terminal shell | :heavy_check_mark: | :soon: | :x: | :soon: |
```
Not sure what "Terminal shell" means exactly, so I guess it is not supported yet.

Comment (sam-golioth) [reply README.md:111]: I _think_ it just means that there is some shell commands for interacting with Golioth.

**Review by mniestroj — APPROVED** [review]
Look good, as long as it will be squashed into single commit to remove noise before merging.

**Pull request merged**